packages/core/src/locales/ar.ts

```diff
@@ -53,66 +53,65 @@ const Nouns: {
     case "invalid_type":
-      return `نوع غير صالح: كان من المتوقع ${issue.expected}، تم استلام ${parsedType(issue.input)}`;
+      return `مدخلات غير مقبولة: يفترض إدخال ${issue.expected}، ولكن ما تم إدخاله ${parsedType(issue.input)}`;
     case "invalid_value":
-      if (issue.values.length === 1)
-        return `القيمة غير صالحة. القيمة المتوقعة ${util.stringifyPrimitive(issue.values[0])}`;
-      return `القيمة غير صالحة: كان من المتوقع أن تكون إحدى القيم التالية:  ${util.joinValues(issue.values, "|")}`;
+      if (issue.values.length === 1) return `مدخلات غير مقبولة: يفترض إدخال ${util.stringifyPrimitive(issue.values[0])}`;
+      return `اختيار غير مقبول: يتوقع انتقاء أحد هذه الخيارات: ${util.joinValues(issue.values, "|")}`;
     case "too_big": {
       const adj = issue.inclusive ? "<=" : "<";
       const sizing = getSizing(issue.origin);
       if (sizing)
-        return `كبير جدًا: من المتوقع أن يحتوي ${issue.origin ?? "value"} على ${adj} ${issue.maximum.toString()} ${sizing.unit ?? "elements"}`;
-      return `كبير جدًا: من المتوقع أن يكون ${issue.origin ?? "value"} ${adj} ${issue.maximum.toString()}`;
+        return ` أكبر من اللازم: يفترض أن تكون ${issue.origin ?? "القيمة"} ${adj}${issue.maximum.toString()} ${sizing.unit ?? "عنصر"}`;
+      return `أكبر من اللازم: يفترض أن تكون ${issue.origin ?? "القيمة"} ${adj}${issue.maximum.toString()}`;
     }
     case "too_small": {
       const adj = issue.inclusive ? ">=" : ">";
       const sizing = getSizing(issue.origin);
       if (sizing) {
-        return `صغير جدًا: من المتوقع أن يحتوي ${issue.origin} على ${adj} ${issue.minimum.toString()} ${sizing.unit}`;
+        return `أصغر من اللازم: يفترض لـ ${issue.origin} أن يكون ${adj}${issue.minimum.toString()} ${sizing.unit}`;
       }
 
-      return `صغير جدًا: من المتوقع أن يكون ${issue.origin} ${adj} ${issue.minimum.toString()}`;
+      return `أصغر من اللازم: يفترض لـ ${issue.origin} أن يكون ${adj}${issue.minimum.toString()}`;
     }
     case "invalid_format": {
       const _issue = issue as errors.$ZodStringFormatIssues;
-      if (_issue.format === "starts_with") return `تنسيق غير صالح: يجب أن يبدأ بـ "${issue}"`;
-      if (_issue.format === "ends_with") return `تنسيق غير صالح: يجب أن ينتهي بـ "${_issue.suffix}"`;
-      if (_issue.format === "includes") return `تنسيق غير صالح: يجب أن يتضمن "${_issue.includes}"`;
-      if (_issue.format === "regex") return `تنسيق غير صالح: يجب أن يطابق النمط ${_issue.pattern}`;
-      return `${Nouns[_issue.format] ?? issue.format} غير صالح`;
+      if (_issue.format === "starts_with") return `نَص خطأ: يجب أن يبدأ بـ "${issue}"`;
+      if (_issue.format === "ends_with") return `نَص خطأ: يجب أن ينتهي بـ "${_issue.suffix}"`;
+      if (_issue.format === "includes") return `نَص خطأ: يجب أن يتضمَّن "${_issue.includes}"`;
+      if (_issue.format === "regex") return `نَص خطأ: يجب أن يطابق النمط ${_issue.pattern}`;
+      return `Invalid ${Nouns[_issue.format] ?? issue.format}`;
```

Comment:
This is not translated.
```suggestion
      return `${Nouns[_issue.format] ?? issue.format} غير مقبول`;
```